Competitive_Programming/C++/Maths & Number Theory/EuclidAlgorithm.md

```diff
@@ -0,0 +1,79 @@
+# Euclid Algorithm
+
+## GCD 
```

Comment:
Please differentiate this two approaches with explanation. Before going to code, the user/reader need to understand the concept/approach that used.

Like: how we're going to solve this problem in this approach.